indico/modules/events/registration/fields/accompanying.py

```diff
@@ -12,10 +12,17 @@
 from indico.core import signals
 from indico.core.marshmallow import mm
 from indico.modules.events.registration.fields.base import RegistrationFormBillableField
+from indico.modules.users.models.users import PersonMixin
 from indico.util.i18n import _
 from indico.util.marshmallow import not_empty
 
 
+class _AccompanyingPerson(PersonMixin):
```

Comment:
would it be ok to make it a public class so I can use it in the cern access plugin as well?

Reply:
sure